scalyr_agent/tests/scalyr_logging_test.py

```diff
@@ -371,6 +374,52 @@ def test_force_valid_metric_or_field_name(self):
         fixed_name2 = scalyr_logging.AgentLogger.force_valid_metric_or_field_name(name2)
         self.assertEqual(fixed_name2, name2)
 
+    def fake_std_write(self, a, b):
+        self.ran_std_output = True
+
+    def test_output_to_file_with_stdout(self):
+        self.ran_std_output = False
+        stdout_bkp = sys.stdout.write
+        sys.stdout.write = types.MethodType(self.fake_std_write, sys.stdout)
+        try:
+            self.__logger.info("Hello world", force_stdout=True)
```

Comment:
It probably also wouldn't hurt to add a test case for negative / default scenario when force arguments are False.